libs/paths/Project.ml

```diff
@@ -1,8 +1,17 @@
-type kind = Git_project | Other_project [@@deriving show]
+(*
+   The type of project.
+
+   A git project must work with the 'git' commands.
+
+   A gitignore project is a fake git project. It doesn't have a valid
```

Comment:
seems weird to have a .gitignore but no .git
Which project have one but not the other?